app/src/main/java/com/yeonae/chamelezone/view/course/CourseModifyActivity.kt

```diff
@@ -53,6 +52,17 @@ class CourseModifyActivity : AppCompatActivity(), CourseModifyContract.View,
         edt_course_title.text = SpannableStringBuilder(courseList[0].title)
         edt_course_content.text = SpannableStringBuilder(courseList[0].content)
 
+        imageContainer.removeAllViews()
+        val rlSlideImg = LayoutInflater.from(this).inflate(
+            R.layout.slider_item_image,
+            imageContainer,
+            false
+        ) as ImageView
+        imageContainer.addView(rlSlideImg)
+        rlSlideImg.image_item.run {
```

Comment:
activity내부에서 item을 만들어서 사용할때, 
findViewById를 사용하는게 좀 더 명시적일 것 같아서 제안해봅니다.

```suggestion
        rlSlideImg.findViewById<ImageView>(R.id.image_item).run {
```

Reply:
수정했습니당
bb5385c